Builder - file name not printed

https://travis-ci.org/CloudSlang/cloud-slang-content/builds/238929220?utm_source=github_status&utm_medium=notification

```console
02/06/17 22:05:30 [INFO] Test case passed: testRemoteSecureCopySucessLocalToRemote [/home/travis/build/CloudSlang/cloud-slang-content/test/io/cloudslang/base/remote_file_transfer/remote_secure_copy.inputs.yaml]. Finished running: io.cloudslang.base.remote_file_transfer.test_remote_secure_copy_local_to_remote with result: SUCCESS
02/06/17 22:05:30 [ERROR]
02/06/17 22:05:30 [ERROR] ------------------------------------------------------------
02/06/17 22:05:30 [ERROR] Exception: Error at line [13] - There should be an empty line between two sections of different tags (@description and @input)
02/06/17 22:05:30 [ERROR] Exception: Error at line [13] - There should be an empty line between two sections of different tags (@description and @input)
02/06/17 22:05:30 [ERROR] Exception: Error at line [11] - There should be an empty line between two sections of different tags (@description and @input)
02/06/17 22:05:30 [ERROR] Exception: Error at line [12] - There should be an empty line between two sections of different tags (@description and @input)
02/06/17 22:05:30 [ERROR] FAILURE: Validation of slang files for project: "/home/travis/build/CloudSlang/cloud-slang-content" failed.
02/06/17 22:05:30 [ERROR] ---------------
```

which file?

--- a/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java
+++ b/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java
@@ -73,13 +73,12 @@ public class SlangContentVerifier {
         loggingService.logEvent(Level.INFO, "");
         Queue<RuntimeException> exceptions = new ArrayDeque<>();
         String errorMessagePrefixMetadata = "";
-        String errorMessagePrefixCompilation = "";
         for (File slangFile: slangFiles) {
             Executable sourceModel = null;
             try {
                 errorMessagePrefixMetadata = "Failed to extract metadata for file: \'" +
                     slangFile.getAbsoluteFile() + "\'.\n";
-                errorMessagePrefixCompilation = "Failed to compile file: \'" +
+                String errorMessagePrefixCompilation = "Failed to compile file: \'" +
                     slangFile.getAbsoluteFile() + "\'.\n";
 
                 Validate.isTrue(slangFile.isFile(), "file path \'" + slangFile.getAbsolutePath() +

--- a/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java
+++ b/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java
@@ -24,9 +24,11 @@ import io.cloudslang.lang.tools.build.validation.MetadataMissingException;
 import io.cloudslang.lang.tools.build.validation.StaticValidator;
 import java.io.File;
 import java.util.ArrayDeque;
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
 import java.util.Queue;
 import java.util.Set;
@@ -70,9 +72,13 @@ public class SlangContentVerifier {
         loggingService.logEvent(Level.INFO, slangFiles.size() + " .sl files were found");
         loggingService.logEvent(Level.INFO, "");
         Queue<RuntimeException> exceptions = new ArrayDeque<>();
+        String errorMessagePrefix = "";
         for (File slangFile: slangFiles) {
             Executable sourceModel = null;
             try {
+                errorMessagePrefix = "Failed to extract metadata for file: \'" +
+                    slangFile.getAbsoluteFile() + "\'.\n";
+
                 Validate.isTrue(slangFile.isFile(), "file path \'" + slangFile.getAbsolutePath() +
                         "\' must lead to a file");
                 SlangSource slangSource = SlangSource.fromFile(slangFile);
@@ -84,7 +90,7 @@ public class SlangContentVerifier {
                 MetadataModellingResult metadataResult = metadataExtractor
                         .extractMetadataModellingResult(slangSource, shouldValidateCheckstyle);
                 Metadata sourceMetadata = metadataResult.getMetadata();
-                exceptions.addAll(metadataResult.getErrors());
+                exceptions.addAll(prependPrefix(metadataResult.getErrors(), errorMessagePrefix));
 
                 if (sourceModel != null) {
                     int size = exceptions.size();
@@ -95,8 +101,7 @@ public class SlangContentVerifier {
                     }
                 }
             } catch (Exception e) {
-                String errorMessage = "Failed to extract metadata for file: \'" +
-                        slangFile.getAbsoluteFile() + "\'.\n" + e.getMessage();
+                String errorMessage = errorMessagePrefix + e.getMessage();
                 loggingService.logEvent(Level.ERROR, errorMessage);
                 exceptions.add(new RuntimeException(errorMessage, e));
                 if (e instanceof MetadataMissingException && sourceModel != null) {
@@ -113,6 +118,18 @@ public class SlangContentVerifier {
         preCompileResult.addExceptions(exceptions);
         preCompileResult.addResults(slangModels);
         return preCompileResult;
+    }
+
+    private Collection<RuntimeException> prependPrefix(Collection<RuntimeException> errors, String prefix) {
+        List<RuntimeException> result = new ArrayList<>();
+        for (RuntimeException ex : errors) {
+            result.add(wrapWithPrefix(ex, prefix));
+        }
+        return result;
+    }
+
+    private RuntimeException wrapWithPrefix(RuntimeException rex, String prefix) {
+        return new RuntimeException(prefix + rex.getMessage(), rex);
     }
 
     public CompileResult compileSlangModels(Map<String, Executable> slangModels) {

--- a/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java
+++ b/cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java
@@ -72,11 +72,14 @@ public class SlangContentVerifier {
         loggingService.logEvent(Level.INFO, slangFiles.size() + " .sl files were found");
         loggingService.logEvent(Level.INFO, "");
         Queue<RuntimeException> exceptions = new ArrayDeque<>();
-        String errorMessagePrefix = "";
+        String errorMessagePrefixMetadata = "";
+        String errorMessagePrefixCompilation = "";
         for (File slangFile: slangFiles) {
             Executable sourceModel = null;
             try {
-                errorMessagePrefix = "Failed to extract metadata for file: \'" +
+                errorMessagePrefixMetadata = "Failed to extract metadata for file: \'" +
+                    slangFile.getAbsoluteFile() + "\'.\n";
+                errorMessagePrefixCompilation = "Failed to compile file: \'" +
                     slangFile.getAbsoluteFile() + "\'.\n";
 
                 Validate.isTrue(slangFile.isFile(), "file path \'" + slangFile.getAbsolutePath() +
@@ -85,12 +88,12 @@ public class SlangContentVerifier {
 
                 ExecutableModellingResult preCompileResult = slangCompiler.preCompileSource(slangSource);
                 sourceModel = preCompileResult.getExecutable();
-                exceptions.addAll(preCompileResult.getErrors());
+                exceptions.addAll(prependPrefix(preCompileResult.getErrors(), errorMessagePrefixCompilation));
 
                 MetadataModellingResult metadataResult = metadataExtractor
                         .extractMetadataModellingResult(slangSource, shouldValidateCheckstyle);
                 Metadata sourceMetadata = metadataResult.getMetadata();
-                exceptions.addAll(prependPrefix(metadataResult.getErrors(), errorMessagePrefix));
+                exceptions.addAll(prependPrefix(metadataResult.getErrors(), errorMessagePrefixMetadata));
 
                 if (sourceModel != null) {
                     int size = exceptions.size();
@@ -101,7 +104,7 @@ public class SlangContentVerifier {
                     }
                 }
             } catch (Exception e) {
-                String errorMessage = errorMessagePrefix + e.getMessage();
+                String errorMessage = errorMessagePrefixMetadata + e.getMessage();
                 loggingService.logEvent(Level.ERROR, errorMessage);
                 exceptions.add(new RuntimeException(errorMessage, e));
                 if (e instanceof MetadataMissingException && sourceModel != null) {

